toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -819,12 +832,49 @@ Additional arguments to pass to the script.
 Example:
 
 ```yaml
-os:
-  postInstallScripts:
+scripts:
+  postCustomization:
   - path: scripts/a.sh
     args: abc
 ```
 
+## scripts type
+
+Note: Script files must be in the same directory or a child directory of the directory
+that contains the config file.
+
+### postCustomization [[script](#script-type)[]]
+
+Scripts to run after all the in-built customization steps have run.
+
+These scripts are run under a chroot of the customized OS.
+
+Example:
+
+```yaml
+scripts:
+  postCustomization:
+  - path: scripts/a.sh
+```
+
+### finalizeCustomization [[script](#script-type)[]]
+
+Scripts to run at the end of the customization process.
+
+These scripts are run under a chroot of the customized OS.
+
+Note: Most scripts should be added to [postCustomization](#postcustomization-script).
```

Comment:
is it possible to clarify what mic runs between postCustomization and finalizeCustomization so that users can make the call on where to place their call?

Reply:
I see it explained clearly in the steps at the top. Maybe a small mention here too - but I'm fine either way.

Reply:
Fixed.